doc/api/deprecations.md

```diff
@@ -3761,6 +3761,19 @@ Instantiating classes without the `new` qualifier exported by the `node:repl` mo
 It is recommended to use the `new` qualifier instead. This applies to all REPL classes, including
 `REPLServer` and `Recoverable`.
 
+### DEP0186: `util.getCallSite`
+
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/55626
+    description: Runtime deprecation.
+-->
+
+Type: Runtime
+
+The `util.getCallSite` API has been removed. Please use [`util.getCallSites()`][] instead.
```

Comment:
Doesn't this have to be doc deprecated first?

Reply:
This is an API in active development status, not stable yet. I don't think it is necessary to document deprecated.

Reply:
We should not deprecate experimental APIs, deprecation is for stable features we want to phase out.

Reply:
There were deprecation for experimental APIs like https://github.com/nodejs/node/blob/main/doc/api/deprecations.md#dep0173-the-assertcalltracker-class.

I was hesistant about adding a deprecation code for this. But all our toolings like `expectWarning` expects a deprecation code.